### ISSUE TITLE
[TTPReq] Education: How to accept an alternative payment method

### ISSUE THREAD

**Issue by staskus** [woocommerce/woocommerce-ios#14165]
Designs & Copy: pdfdoF-5KF-p2
Flows: pdfdoF-5GF-p2
Depends on: https://github.com/woocommerce/woocommerce-ios/issues/14159

## Education Step Screen Functionality

If it's not already done, I suggest creating a reusable full-screen scroll view to fit as much educational content as needed. Support text blocks, or numbered text blocks as we have in 'Set up Tap to Pay'.

##  How to accept an alternative payment method

- Only integrate within required markets (UK)
- If an image is not yet available, use a placeholder image

## Designs

![Image](https://github.com/user-attachments/assets/aaec9c1f-26e1-46cb-89df-d94818aae654)

**Comment by dangermattic** [comment]
Thanks for reporting! 👍
